packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/getLocationDetailViewTableData/getLocationDetailViewTableData.ts

```diff
@@ -17,6 +17,7 @@ export const getLocationDetailViewTableData = ({
   location,
   fileDataItems,
   hasFiles,
+  isLoading,
```

Comment:
Should be able to remove this and use context as well here (and anywhere else with the same getXTableData pattern)

Reply:
Yep, this is right. Updated to remove here and like locations